Makefile

```diff
@@ -10,9 +10,9 @@ TASK = $(TOOL_DIR)/task
 ## Bootstrapping targets #################################
 
 $(BINNY):
-	@mkdir -p $(TOOL_DIR)
-	@# we don't have a release of binny yet, so build off of the current branch
-	@#curl -sSfL https://raw.githubusercontent.com/$(OWNER)/$(PROJECT)/main/install.sh | sh -s -- -b $(TOOL_DIR)
+	@-mkdir $(TOOL_DIR)
```

Comment:
The `-` preceding the command is to prevent it exiting on failure. Windows doesn't support the `-p`, instead creating a directory with that name. In this case, we only need a directory 1-level deep, so we don't really need `-p`. Maybe a better idea would be to just add a file to the repo at `.tool/empty` and get rid of creating this directory altogether?